src/components/grid/grid.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import classNames from 'classnames';
 import GridIcon from './grid_icon';
 import GridLabel from './grid_label';
+import { Link } from 'react-router';
```

Comment:
Agree! That's much more robust.

I've fixed it to provide `component` property to be set in Grid as similar to `Cell` class.